src/Hack.sol

```diff
@@ -457,8 +453,6 @@ contract GuessTheAverage {
     mapping(address => Player) public players; // Maps an address to its respective Player status.
     mapping(uint256 => address) public indexToPlayer; // Maps a guess index to the player who made the guess.
 
-    /* CONSTRUCTOR */
-
     constructor(uint32 _commitDuration, uint32 _revealDuration) {
```

Comment:
```suggestion
    constructor(uint32 commitDuration, uint32 revealDuration) {
```

Reply:
On all the exercise. It is the same for the exercise 10.

Reply:
Yes that is the tricky case. @MerlinEgalite idk how to name the variable and the parameter in this case (when the variable must be immutable and public): 

```
uint256 public immutable someVariable;

constructor(uint256 _someVariable) {
    someVariable = _someVariable;
}
```